### ISSUE TITLE
Add some links in footer section

### ISSUE THREAD

**Issue by ROHAN842** [CodeFlow201/codeflow#88]
The footer section doesn't consists of About, Team and Contact Us page options as well as their relevant links for different pages.
I would like to add these links in the footer section.

I would like to work on this issue.

Kindly assign this issue to me under LGMSOC.

**Comment by ROHAN842** [comment]
Hey @hkaur008 please assign this issue to me.

**Comment by ROHAN842** [comment]
@iamparasbansal sir and @vaibhavarora102 sir please assign this issue to me

**Comment by d-cryptic** [comment]
Can I work on this issue under hacktoberfest??

**Comment by KarishmaVanwari** [comment]
Hey @hkaur008 can I work on this issue?

**Comment by hkaur008** [comment]
@d-cryptic  would you like have this assigned ?

**Comment by hkaur008** [comment]
Yes, we need to at QR code to codeflow linkedIn and  fix extra space of last links of footer

As they are not in same line

**Comment by khushbooag4** [comment]
![image](https://user-images.githubusercontent.com/88361750/136645066-46e8662c-8d0b-4832-9e2d-c2312671c758.png)
Any suggestions??

**Comment by d-cryptic** [comment]
I think the dark background should be on 'social media links' and 'other links'

**Comment by hkaur008** [comment]
I m thing to improve cards we can surely change this later wards

**Comment by khushbooag4** [comment]
I have created a pull request. @hkaur008 can you review it
